lib/puppetserver/ca/action/list.rb

```diff
@@ -209,8 +210,9 @@ def separate_certs(all_certs)
           return requested, signed, revoked
         end
 
-        def get_all_certs(settings)
-          result = Puppetserver::Ca::CertificateAuthority.new(@logger, settings).get_certificate_statuses
+        def get_all_certs_or_csr(settings, queried_state = nil)
```

Comment:
`all` still seems a little weird, since you can filter by state. maybe just `get_certs_or_csrs`?